Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -546,9 +546,8 @@ QProgressBar::chunk {
     min-height:24px;
 }
 
-#GemCatalogHeaderLabel {
-    font-size: 12px;
-    color: #FFFFFF;
+#adjustableHeaderWidget QHeaderView::section { 
+    background-color:#333333;
```

Comment:
Nitpick:  I think you could actually use `transparent` for the color here, instead of matching the background color, which would be future proof if we decide to change the default background color.